lib/interactive_viewer.ml

```diff
@@ -9,13 +9,44 @@ let current_patch patches =
   let$ index = index in
   List.nth_opt patches index
 
-let string_of_operation = Format.asprintf "%a" (Patch.pp_operation ~git:false)
+let string_of_operation operation =
+  match operation with
+  | Patch.Create path ->
+    Ui.hcat [
+          W.string "Creation of ";
+          W.string ~attr:Notty.A.(fg green ++ st bold) ("" ^ path ^ "")
+    ]
+  | Patch.Delete path ->
+    Ui.hcat [
+      W.string "Deletion of ";
+      W.string ~attr:Notty.A.(fg red ++ st bold) ("" ^ path ^ "")
+    ]
+  | Patch.Rename (old_path, new_path) ->
+    Ui.hcat [
+      W.string "Rename with modifications ";
+      W.string ~attr:Notty.A.(fg blue ++ st bold) ("" ^ old_path ^ "");
+      W.string " to ";
+      W.string ~attr:Notty.A.(fg green ++ st bold) ("" ^ new_path ^ "");
+    ]
+  | Patch.Rename_only (old_path, new_path) ->
+    Ui.hcat [
+      W.string "Rename ";
+      W.string ~attr:Notty.A.(fg blue ++ st bold) ("" ^ old_path ^ "");
+      W.string " to ";
+      W.string ~attr:Notty.A.(fg green ++ st bold) ("" ^ new_path ^ "");
+    ]
+  | Patch.Edit path ->
+    Ui.hcat [
+      W.string "Modification of ";
+      W.string ~attr:Notty.A.(fg red ++ st bold) ("" ^ path ^ "")
```

Comment:
I would have chosen `blue` for the edit color: `red` makes it seem to have been deleted...